test/test-framework/pkg/apis/cache/v1alpha1/zz_generated.openapi.go

```diff
@@ -0,0 +1,221 @@
+// +build !ignore_autogenerated
```

Comment:
Can you remove this file? The SDK doesn't use this generated code.

Side note: I'm considering removing the generator for this file, since most SDK users do not use it, or at least make it optional.

Reply:
To remove it we should update the annotations `// +k8s:openapi-gen=true` to false. I will do that 👍

Reply:
For now we can keep it here. If we decide that removing the openapi generator is the right way to go we can remove this file.